Update controls overview and other articles where applicable (#662)
telerik/maui-docs: controls/slideview/overview.md

# .NET MAUI SlideView Overview

The Telerik UI for .NET MAUI SlideView is a flexible navigation control that allows you to slide between different views, thus providing an interactive navigation. With SlideView, you can build a gallery display to show your images and content efficiently.

![.NET MAUI SlideView Overview](images/slideview-overview.png)

## Key Features

> * [Navigation]({%slug slideview-interaction%})&mdash;The SlideView control provides variety of navigation through it`s items - navigation buttons, indicators(Desktop) and through pan gesture.

Reviewer: ```suggestion
* [Navigation]({%slug slideview-interaction%})&mdash;The SlideView control provides a variety of ways for navigation through its items—navigation buttons, indicators (Desktop), and pan gestures.
```

* [Buttons]({%slug slideview-interaction%})&mdash;Users can use the navigation buttons to switch through the different views.

> * [Indicators]({%slug slideview-indicators%})&mdash;Show changes when navigation from the current item to another item.

Reviewer: ```suggestion
* [Indicators]({%slug slideview-indicators%})&mdash;Show changes when navigating from the current item to another item.
```
